config/plugins/visualizations/fits_graph_viewer/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "fits-graph-visualization",
+    "version": "1.0.0",
+    "keywords": [
+        "galaxy",
+        "visualization"
+    ],
+    "main": "index.js",
+    "license": "AFL-3.0",
```

Comment:
Shouldn't this be `MIT` and not `AFL-3.0`?

Reply:
@francoismg , why did you select `AFL-3.0`?

Reply:
As per the last sentence by @francoismg  in https://github.com/galaxyproject/galaxy/pull/19003#issue-2591306274

> * [x]  I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

 I'm changing this to `MIT` in https://github.com/galaxyproject/galaxy/pull/20264